llvm/test/CodeGen/AMDGPU/sdwa-peephole-cndmask-vop2.mir

```diff
@@ -0,0 +1,41 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -run-pass=si-peephole-sdwa -mcpu=gfx803 -o - %s | FileCheck  -check-prefix=GFX8 %s
+# RUN: llc -mtriple=amdgcn-amd-amdhsa -run-pass=si-peephole-sdwa -mcpu=gfx1100 -o - %s | FileCheck -check-prefix=GFX11 %s
+
+---
```

Comment:
This test should merge into the other 2. This also should really have a separate wave32 and wave64 version, and it's a bug you need to use vcc in wave32

Reply:
> This test should merge into the other 2. This also should really have a separate wave32 and wave64 version, and it's a bug you need to use vcc in wave32

Ok, I will merge the tests. Concerning the bug: You mean having to specify `implicit $vcc` in ` %4:vgpr_32 = V_CNDMASK_B32_e32 killed %2, killed %3, implicit $exec, implicit $vcc` with "wave32"?

Reply:
It should be vcc_lo. Ideally the instruction definitions would be swapped out depending on HwMode

Reply:
> This test should merge into the other 2. 
Done ...

> This also should really have a separate wave32 and wave64 version, and it's a bug you need to use vcc in wave32
 ... and this is an unrelated/follow-up issue if I understand this correctly.